### PR TITLE
App modifications

### PR DESCRIPTION
**Pull request by S-Dafarra**
This PR extracts the application modifications from https://github.com/robotology/walking-teleoperation/tree/modify_dcmWalkingRetargeting_rpc_Oculus

### CODE REVIEW

**Comment by S-Dafarra** [comment]
The audio still need to be checked

**Review by isorrentino — APPROVED** [review]

**Comment by S-Dafarra** [comment]
This is now blocked by #76

File: app/scripts/Xprize-VisualRetargeting.xml
Comment (kouroshD) [inline app/scripts/Xprize-VisualRetargeting.xml:35]: Probably, here we can use `--from dcmWalkingRetargeting.ini ` instead of this long line of list of parameters.
Should be checked, if all the variables are correctly set then in the `.ini` file.

**Review by kouroshD — COMMENTED** [review]
Thank you @S-Dafarra  for the PR. 
Two small points to be considered before merging.

**Comment by kouroshD** [comment]
The new file names related to the `JoypadControlServer` device parameters is called `dcmWalkingRetargeting.ini`. Probably, it is copy-pasting problem, since the configuration file name is misleading. Probably, sth like `JoypadControlServerParameters.ini` can be better representative of those parameters.

File: app/scripts/Xprize-VisualRetargeting.xml
Comment (kouroshD) [inline app/scripts/Xprize-VisualRetargeting.xml:73]: Why we are changing it? If not wrong, `mjpeg` doing some compression of the data before buffering it to the network. Normally, for images, `mjpeg`is recommended to be used.

File: app/scripts/Xprize-KynDynRetargeting.xml
Comment (kouroshD) [inline app/scripts/Xprize-KynDynRetargeting.xml:24]:
```suggestion
    <parameters>--use_transform_server true</parameters>
    <dependencies>
          <port timeout="15.0">/transformServer/transforms:o</port>
     </dependencies>
```

Comment (kouroshD) [reply app/scripts/Xprize-KynDynRetargeting.xml:24]: @S-Dafarra  I've added this dependency to the transform server port, since with the proposed option we have dependency on it. If it is OK for you, please consider committing it.

Comment (S-Dafarra) [reply app/scripts/Xprize-KynDynRetargeting.xml:24]: Good idea! I just removed an extra space.

**Comment by kouroshD** [comment]
I've rebased the branch after #76  and #73 .

**Pull request merged**
